### ISSUE TITLE
Checkbox doesn't have "Check" Icon in it?

### ISSUE THREAD

**Issue by TheStarkster** [antvis/component#324]
I was rendering checkbox component in my AntV S2 table and i see when the checkbox is checked, there's just color fill, but no icon in it, Is that normal? Or am i doing something wrong?

<img width="153" alt="Screenshot 2024-08-24 at 2 11 32 PM" src="https://github.com/user-attachments/assets/47716839-1209-4874-a3c7-119fd6c36701">

```typescript
this.append(new Checkbox({
                style: {
                    boxX: this.meta.x + 10,
                    boxY: this.meta.y + 10,
                    checked: this.selectedRows?.includes(Number(this.meta.rowIndex)),
                    checkedX: this.meta.x + 10,
                    checkedY: this.meta.y + 10,
                },
            }))
```

**Comment by TheStarkster** [comment]
On further investigation it seems no property starting with prefix "checked" are working checkedX, checkedY etc. but individual boolean property checked works fine as you can also see in the screenshot in the comment before

**Comment by hustcc** [comment]
Usage reference from [antd](https://ant-design.antgroup.com/components/checkbox-cn).

**Comment by TheStarkster** [comment]
> no icon

I meant there's no "check" within the checkbox, notice in the screenshot i've provided, it's just blue box there's suppose to be a white check

**Comment by hustcc** [comment]
yes, we have no check shape. Are you interested in taking on this task with a pr?

![image](https://github.com/user-attachments/assets/2c8f87bd-fa20-4a0a-9b0f-e263caa36d08)

**Comment by TheStarkster** [comment]
@hustcc sure, in-fact I fixed it already, Giving a PR shortly have a look👇


![Before after minimal skincare Instagram post](https://github.com/user-attachments/assets/b2fb7da8-ecfa-4eab-a4f6-d9b206b89721)
